assets/styles/homepage.scss

```diff
@@ -206,10 +214,12 @@ h2 {
 
 .section-footer-links {
   @include padding(2rem null);
-
-  img {
-    margin-right: 1rem;
-    max-width: 3.4rem;
+  a {
+      margin-right: 1rem;
+      
+    img {
```

Comment:
I think we can de-nest the img so that it passes are linter rule max of 2 levels.